blocks/columns/columns.css

```diff
@@ -19,15 +15,98 @@
   display: block;
 }
 
-@media (min-width: 900px) {
-  .columns > div {
-    align-items: center;
-    flex-direction: unset;
-    gap: 32px;
+/* section metadata */
+main .section.highlight {
```

Comment:
So - for this, you don't want to put anything in a block level CSS that isn't scoped to the block. All of your CSS selectors should start with the block.

Personally, to be explicit, i like to start with 

```
.<blockname>.block
```

So it's clear that you're scoping to this context.